hydra-node/src/Hydra/Options.hs

```diff
@@ -469,16 +477,14 @@ toArgs
       Testnet (NetworkMagic magic) -> show magic
 
     argsChainConfig =
-      mempty
-        <> ["--network-id", toArgNetworkId networkId]
+      ["--network-id", toArgNetworkId networkId]
```

Comment:
You're ruining @KtorZ's efforts towards a more aligned world!

Reply:
It wasn't me.. it was `hlint` :speak_no_evil: